frontend/public/components/factory/table.tsx

```diff
@@ -494,8 +495,9 @@ const getActiveColumns = (
   componentProps: ComponentProps,
   activeColumns: Set<string>,
   columnManagementID: string,
+  t: any,
```

Comment:
We should use the `TFunction` type

```suggestion
  t: TFunction,
```

Reply:
I'm a little unsure about passing `t` around as a function parameter. It might be better to access the global `i18n` instance directly as long as it will rerender anyway on language changes. I think the case we need to be careful about is module level `const` variables that don't ever get reevaluated.

Reply:
used global t function instead of passing t, and updated type